src/shared.ts

```diff
@@ -143,16 +143,28 @@ export const ESGlobalKeys = SetCreate([
     'Intl', // Unstable
 ]);
 
-// These are foundational things that should never be wrapped but are equivalent
+// These are intrinsics that can be reached by syntax, and must be mapped between realms.
 // TODO: revisit this list.
-export const ReflectiveIntrinsicObjectNames = [
-    'Object',
-    'Function',
-    'URIError',
-    'TypeError',
-    'SyntaxError',
-    'ReferenceError',
-    'RangeError',
-    'EvalError',
-    'Error',
-];
+export function extractUndeniableIntrinsics(globalObj: typeof globalThis): any[] {
+    return map(globalObj.eval(`[
+        ({}),
+        (_=>1),
+        [],
+        /x/,
+        true,
+        (1),
+        "",
+        (async Promise=>1)(),
+        (async AsyncFunc=>1),
+        (function* GeneratorFunc(){}),
+        (async function* AsyncGeneratorFunc(){}),
+        // TODO: Errors as undeniable must be reviewed to avoid global lookup
+        new URIError,
+        new TypeError,
+        new SyntaxError,
+        new ReferenceError,
+        new RangeError,
+        new EvalError,
+        new Error,
+    ]`), (o: any) => o.__proto__);
```

Comment:
I think we have a `getPrototypeOf` helper which could make the map transform `, getPrototypeOf);`